master/master.cfg

```diff
@@ -56,12 +58,17 @@ c = BuildmasterConfig = {}
 
 # horizons
 c['changeHorizon'] = 300
-c['buildHorizon'] = 300
-c['eventHorizon'] = 300
-c['logHorizon'] = 300
 
-# slaves are set up in local.py
-c['slaves'] = SLAVES
+# configure a janitor which will delete all logs older than one month,
+# and will run on sundays at noon
+c['configurators'] = [util.JanitorConfigurator(
+    logHorizon=timedelta(weeks=4),
+    hour=12,
+    dayOfWeek=6
+)]
```

Comment:
I don't really like that this is listed as a "builder".  Is there any way around that?

Reply:
@tardyp at http://buildbot.python.org/test/#/builders and http://buildbot.python.org/test/#/workers, `__Janitor` shows up as a builder.  @zware doesn't like this.  Is there a way to stop having `__Janitor` appear as a builder?  
I can't figure out a way to do this.

Reply:
For now there is no way. This as been discussed several time over IRC, but I can't find an issue about it. What was originally discussed is to just hide everything that starts with '__', but this is not yet implemented. I think this is pretty cosmetic, but I am open to discussion.

Reply:
https://github.com/buildbot/buildbot/issues/3677